spec/smartcar/helpers/auth_helper.rb

```diff
@@ -37,8 +37,8 @@ def run_auth_flow(authorization_url, test_email = nil, make = nil)
       driver = Selenium::WebDriver.for(:firefox, capabilities: [options])
       driver.navigate.to authorization_url
       driver.manage.timeouts.implicit_wait = 10
-      driver.find_element(:css, 'button#continue-button').click
-      driver.find_element(css: "button[id=\"#{make}\"]").click
+      driver.find_element(css: 'button#continue-button').click
+      driver.find_element(css: "button##{make}.brand-list-item").click
```

Comment:
was this one needed ? We had recently fixed it looking at what we do in CLI.  I am fine either way there are probably multiple ways to find the element